images/bind9/root/etc/bind/named.conf.options

```diff
@@ -32,7 +32,7 @@ options {
 	   cause your server to become part of large scale DNS amplification
 	   attacks. Implementing BCP38 within your network would greatly
 	   reduce such attack surface
-	*/
+	*/;
```

Comment:
hi @jadhaj i believe the error is not here, but in the content of the file at [line 24](https://gitlab.cee.redhat.com/aosqe/rdu2-internal/-/blob/master/butane/bastion/bind9.bu?ref_type=heads#L83)